test/cadet/jobs/autograder/lambda_worker_test.exs

```diff
@@ -95,21 +120,6 @@ defmodule Cadet.Autograder.LambdaWorkerTest do
         end
       end
     end
-
-    test "lambda errors", %{question: question, answer: answer} do
```

Comment:
We should still test the case when AWS Lambda crashed

Reply:
i added a test for that case